Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -1,5 +1,5 @@
-import demistomock as demisto
-from CommonServerPython import *
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
```

Comment:
is this change required? It worked before without these.

Reply:
Actually I didn't add it, but I see the master does not contain those lines, so I'll revert it